ix-dev/community/factorio/app.yaml

```diff
@@ -0,0 +1,43 @@
+app_version: v1.0
+capabilities:
+- description: Factorio is able to change file ownership arbitrarily
+  name: CHOWN
+- description: Factorio is able to bypass file permission checks
+  name: DAC_OVERRIDE
+- description: Factorio is able to bypass permission checks for file operations
+  name: FOWNER
+- description: Factorio is able to change group ID of processes
+  name: SETGID
+- description: Factorio is able to change user ID of processes
+  name: SETUID
+categories:
+- games
+changelog_url: https://github.com/factoriotools/factorio-docker
+date_added: '2025-06-20'
+description: The dockerized Factorio gameserver, so the factory can grow even further.
+home: https://github.com/factoriotools/factorio-docker
+host_mounts: []
+icon: 
+keywords:
+- games
+- server
+- factorio
+lib_version: 2.1.35
+lib_version_hash: 1bd4e0058fbd4d7c207df2cae606580065e8e6dba3e232f41bc1b006848b05d2
+maintainers:
+- email: dev@ixsystems.com
+  name: truenas
+  url: https://www.truenas.com/
+name: factorio
+run_as_context:
+- description: factorio Server runs as root user.
+  gid: 845
+  group_name: factorio
+  uid: 845
+  user_name: factorio
```

Comment:
I don't think thats true, you haven't done `c1.set_user()` and docker by default runs as root.